package.json

```diff
@@ -84,13 +87,12 @@
     "url": "http://github.com/Azure/ms-rest-js/issues"
   },
   "scripts": {
-    "build": "run-p build:scripts build:node build:es build:browser",
+    "build": "run-p build:scripts build:lib",
     "build:scripts": "tsc -p ./.scripts/",
-    "build:node": "tsc",
-    "build:es": "tsc -p tsconfig.es.json",
-    "build:browser": "webpack && node node_modules/uglify-es/bin/uglifyjs --source-map -c -m -o msRestBundle.min.js msRestBundle.js",
-    "watch:node": "tsc -w",
-    "watch:browser": "webpack -w",
+    "build:lib": "run-s build:tsc build:rollup build:minify-browser",
+    "build:tsc": "tsc -p tsconfig.es.json",
+    "build:rollup": "rollup -c rollup.config.js",
+    "build:minify-browser": "uglifyjs -c -m --comments --source-map \"content='./dist/msRest.browser.js.map'\" -o ./dist/msRest.browser.min.js ./dist/msRest.browser.js",
```

Comment:
Changing from msRestBundle.js to msRest.browser.js is a breaking change, right? Shouldn't we bump the major version for this?